Desktop/SingleTooltip.scss

```diff
@@ -0,0 +1,9 @@
+.component {
```

Comment:
@daniloprates why do we have this `Desktop/SingleTooltip.scss` file?

Reply:
I meant to move it to my Desktop, not to create a directory 😅